hummingbot/strategy/strategy_base.pxd

```diff
@@ -0,0 +1,8 @@
+from wings.time_iterator cimport TimeIterator
+
+
+cdef class StrategyBase(TimeIterator):
+    cdef active_orders(self)
```

Comment:
All Cython functions should have the `c_` prefix.

If you want a property, then it should be a regular Python function like this.

```
@property
def active_orders(self) -> List[...]:
  ...
```

Reply:
done